packages/components/src/controllers/loading-controller.ts

```diff
@@ -0,0 +1,23 @@
+import type { ReactiveController, ReactiveControllerHost } from './controller-host';
+
+export class LoadingController implements ReactiveController {
```

Comment:
May something like `InitialLoadingController` would be more specific since it is only used for this particular use case

Reply:
Renamed.